app/src/system/java/org/mozilla/tv/firefox/ext/EngineView.kt

```diff
@@ -39,11 +39,12 @@ fun EngineView.setupForApp() {
     // WebView can be null temporarily after clearData(); however, activity.recreate() would
     // instantiate a new WebView instance
     webView?.setOnFocusChangeListener { _, hasFocus ->
-        if (hasFocus) {
-            // See FocusedDOMElementCacheInterface for details on why this happens
-            //
-            // Note that caching of this value is done elsewhere
+        if (!hasFocus) {
```

Comment:
Hm, does this correlate with the comment changes above? This doesn't look like when the app is going to sleep or backgrounded. Are they unrelated changes?

Reply:
The WV loses focus when the app is backgrounded, so it handles that case as well.  I expanded the comments to make this more clear.